src/loaddata.sh

```diff
@@ -13,7 +13,9 @@ then
 echo ${environment} environment
 docker cp $1 ${container}:/app/dump.json
 docker exec -it ${container} /bin/bash -c ". /entrypoint && LOADING=true ./manage.py loaddata /app/dump.json"
+docker exec ${container} /bin/bash -c "rm -rf /app/dump.json"
```

Comment:
```suggestion
docker exec ${container} /bin/bash -c "rm /app/dump.json"
```
-rf shouldn't be necessary since we're deleting a file rather than a directory and there's no need to force it.